src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -4,6 +4,8 @@
 import seedu.addressbook.data.person.ReadOnlyPerson;
 import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
 
+import java.util.List;
```

Comment:
It'll be better to put this statement: import java.util.List; before the statements: import seedu.addressbook... to maintain consistent ordering of import statements

Reply:
you can get the IDE to do these things for you :slightly_smiling_face: